Sources/AblyChat/Messages.swift

```diff
@@ -29,15 +107,43 @@ public struct SendMessageParams: Sendable {
     }
 }
 
+/**
+ * Options for querying messages in a chat room.
+ */
 public struct QueryOptions: Sendable {
     public enum OrderBy: Sendable {
         case oldestFirst
         case newestFirst
     }
 
+    /**
+     * The start of the time window to query from. If provided, the response will include
+     * messages with timestamps equal to or greater than this value.
+     *
+     * Defaults to the beginning of time.
+     */
     public var start: Date?
+
+    /**
+     * The end of the time window to query from. If provided, the response will include
+     * messages with timestamps less than this value.
+     *
+     * Defaults to the current time.
+     */
     public var end: Date?
+
+    /**
+     * The maximum number of messages to return in the response.
+     *
+     * Defaults to 100.
+     */
     public var limit: Int?
+
+    /**
+     * The direction to query messages in.
+     * If ``OrderBy/oldestFirst``, the response will include messages from the start of the time window to the end.
+     * If ``OrderBy/newestFirst``, the response will include messages from the end of the time window to the start.
+     */
     public var orderBy: OrderBy?
```

Comment:
we should include the default direction in this comment

Reply:
Yes, but I don't see the default setting of this in the code. Should be on the server then? Also this convo - https://github.com/ably/ably-chat-swift/pull/185#pullrequestreview-2489260118

Reply:
It is from the server in that if the server receives a query without a direction then there will be a default direction, which is newestFirst. 

<img width="883" alt="Screenshot 2024-12-10 at 15 43 02" src="https://github.com/user-attachments/assets/5025f64b-32b8-4d0c-979b-23f585b805bd">

Reply:
updated